template/build.cmd

```diff
@@ -0,0 +1,17 @@
+@ECHO off
+SETLOCAL
+
+SET BUILD_FOLDER=.build
+
+SET BOOTSTRAP_FILE_REMOTE=https://raw.githubusercontent.com/aspnet/KoreBuild/victorhu/init/bootstrap/build.cmd
+SET BUILD_FILE=%BUILD_FOLDER%\build.cmd
+
+SET REPO_FOLDER=%~dp0
+CD "%REPO_FOLDER%"
+
+if EXIST %BUILD_FILE% GOTO :EndBootstrapGet
```

Comment:
Minor but for simple cases that don't include `SET` commands, an `IF` block is easier to read.